includes/admin/class-wcmp-admin.php

```diff
@@ -539,6 +550,89 @@ public static function getTrackTraceUrl($order_id, $track_trace)
         return $trackTraceUrl;
     }
 
+    /**
+     *
```

Comment:
Deze dockblock is leeg, je kan hier altijd neerzetten wat er in de method gebeurd